ecs_windows/README.md

```diff
@@ -0,0 +1,181 @@
+## Using default Amazon ECS configurations for Windows images
```

Comment:
Can we call the directory storing these configs `ecs_windows_forward_daemon` and then make it clear in title this is for running Fluent Bit as a daemon in ECS using forward plugin. 

And then as soon as the blog launches we link it here?

And then in the future, we can easily just have another dir: `ecs_windows_tail_daemon`

Reply:
That's a good idea. I have made the changes here.